docs/contributing/Local_Development.mdx

```diff
@@ -86,7 +86,19 @@ You can run `yarn typecheck` in any package or in the root to run `tsc`.
 ## Website Development
 
 Our interactive documentation website is built with [Docusaurus](https://docusaurus.io).
-Running `yarn start` from either the root or `packages/website` will start a local dev server on `localhost:3000`.
+After cloning the repository, run the following steps to install and build the website:
+
+```sh
+yarn install # To ensure that the dependencies are installed.
+yarn build # This builds all the packages except for `website-eslint` and `website`.
+cd packages/website-eslint
+yarn build
+cd ../..
+cd packages/website
+yarn build
+cd ../..
+```
+Next, running `yarn start` from either the root or `packages/website` will start a local dev server on `localhost:3000`.
```

Comment:
I think we just need to edit the notes below:

> The `website` package relies on other packages being built.
> We recommend running `yarn build` from the root, `packages/website-eslint`, and from this package  before `yarn start`.

Reply:
yeah thanks